test/saithrift/src/switch_sai_rpc_server.cpp

```diff
@@ -2386,25 +2386,28 @@ class switch_sai_rpcHandler : virtual public switch_sai_rpcIf {
                   attr_list[i].value.u8 = attribute.value.u8;
                   break;
               case SAI_MIRROR_SESSION_ATTR_VLAN_TPID:
-                  attr_list[i].value.u16 = attribute.value.u16;
+                  attr_list[i].value.u16 = attribute.value.u32;
```

Comment:
On client, TPID has to be stored using u32 because:
1. u16 is implemented using i16 on client, which has the range of s16
2. One common value for TPID is 0x8100, which is out of the range of s16

So TPID is now stored using u32 on client instead.